script/utils.js

```diff
@@ -0,0 +1,11 @@
+const { spawn } = require('child_process');
+
+const runCommand = cmd => {
```

Comment:
The idea behind this is so we can run arbitrary shell commands without resorting to a shell script. Because, let's be honest, they're kinda gross.